tests/test_export_onnx.py

```diff
@@ -0,0 +1,243 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import io
+import os
+import unittest
+import warnings
+from functools import wraps
+import onnx
+import torch
+from torch import nn
+from torch._C import OptionalType
+from torch.onnx import register_custom_op_symbolic, unregister_custom_op_symbolic
+from torch.onnx.symbolic_registry import is_registered_op
+
+from detectron2 import model_zoo
+from detectron2.config import get_cfg
+from detectron2.export import add_export_config
+from detectron2.export.flatten import TracingAdapter
+from detectron2.export.torchscript_patch import patch_builtin_len
+from detectron2.layers import ShapeSpec
+from detectron2.modeling import build_model
+from detectron2.modeling.roi_heads import KRCNNConvDeconvUpsampleHead
+from detectron2.structures import Boxes, Instances
+from detectron2.utils.env import TORCH_VERSION
+from detectron2.utils.testing import get_sample_coco_image, random_boxes
+
+SLOW_PUBLIC_CPU_TEST = unittest.skipIf(
+    os.environ.get("CI") and not torch.cuda.is_available(),
+    "The test is too slow on CPUs and will be executed on CircleCI's GPU jobs.",
+)
+
+NOT_IMPLEMENTED_YET = unittest.skipIf(
+    True,
+    "Test is still in development and not stable for running.",
+)
+
+################################################################################
+# Fake symbolics to allow exporting to succeed
```

Comment:
I'd convert this to draft until https://github.com/pytorch/pytorch/issues/27212 is fixed.
We should add grid_sampler export to PyTorch before asking for review of this PR so that fake symbolics are not needed, and we can actually successfully export.

Reply:
I was actually trying to run all pipelines only, but apparently it needs maintainer approval. I was thinking in having a "@skipIfNoGridSampler" decorator to keep the test. This way, it would contain the same tests as the torch scripting component.